Microsoft Teams - Fix issue when trying to find a team from a huge list of teams (#399)
rapid7/insightconnect-plugins: microsoft_teams/help.md

# Version History

> * 2.0.0 - Fix issue where send message would not work if there were too many teams. Removed regex capability for team and channel inputs on most actions. This will speed up 'Send Message' and 'Send HTML Message' 

Reviewer: ```suggestion
* 2.0.0 - Fix issue where send message would not work if there were too many teams | Removed regex capability for team and channel inputs which will speed up Send Message and Send HTML Message actions
```
